### DISCUSSION TITLE
last page read

### DISCUSSION THREAD

**Discussion by barp27** [IReaderorg/IReader#63] [Q&A] [answered]
Hi, the app doesn't really save the last page I have read. I tried that with both modes (incognito mode). 
what can I do?

**Comment by barp27** [comment]
also im trying to download a repo from the extansions, and i get this error: 
![WhatsApp Image 2022-10-19 at 18 11 31](https://user-images.githubusercontent.com/116179917/196733620-fbacd8db-5f0c-446a-a268-97b38e40179b.jpeg)

**Reply by kazemcodes** [reply, marked as answer]
This issue is probably fixed right now ,
Sorry for late reply, I just see this discussion
